packages/quantic/cypress/e2e/refine-toggle/refine-toggle.cypress.ts

```diff
@@ -378,6 +423,49 @@ describe('quantic-refine-toggle', () => {
           });
         });
       });
+
+      describe('when a filter is selected and the search query does not return any results', () => {
```

Comment:
Is this the same test twice?
All I see is `addFacets(WithoutInputs?)();`

Reply:
It's the same test but in two different situations cause the component is supposed to behave in a slightly different way when being displayed with facets with inputs comparing to when being displayed with facets without inputs.